shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -742,9 +742,8 @@ protected void onAttachedToWindow() {
         super.onAttachedToWindow();
         mAccessibilityEnabled = mAccessibilityManager.isEnabled();
         mTouchExplorationEnabled = mAccessibilityManager.isTouchExplorationEnabled();
-        if (mAccessibilityEnabled || mTouchExplorationEnabled) {
-            ensureAccessibilityEnabled();
-        }
+        if (mAccessibilityEnabled || mTouchExplorationEnabled) ensureAccessibilityEnabled();
```

Comment:
Looks like everywhere else in this file we are using if with {} instead of these one-liner. Maybe change it to be consistent?

Reply:
Fixed